README.md

```diff
@@ -277,11 +228,10 @@ I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT.md] licensed.
```

Comment:
- [ ] Kindly modify this license link to MV to fit your project   [mit licensense](https://github.com/microverseinc/readme-template/blob/master/MIT.md)